app/models/user.rb

```diff
@@ -0,0 +1,30 @@
+require 'bcrypt'
+
+class User
+  include DataMapper::Resource
+
+  property :id,        Serial
+  property :name,      String
+  property :username,  String, :required => true, :unique => true, :messages =>
+                               { :presence => 'Please enter a username',
+                                 :is_unique => 'We already have that username' }
+  property :email,     String, :required => true, :unique => true, :format => :email_address,
+                               :messages => { :presence => 'Please enter an email address',
+                                             :is_unique => 'We already have that email',
+                                             :format => 'Please enter a valid email address' }
```

Comment:
Metrics/LineLength: Line is too long. [94/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)